README.md

```diff
@@ -49,6 +49,7 @@ NIPs stand for **Nostr Implementation Possibilities**. They exist to document wh
 | 44          | Channel Mute User           | [28](28.md)            |
 | 45-49       | Public Chat Reserved        | [28](28.md)            |
 | 22242       | Client Authentication       | [42](42.md)            |
+| 1000-9999   | Regular Events Reserved     | [16](16.md)            |
```

Comment:
What's a Regular Event?

Reply:
According to https://github.com/nostr-protocol/nips/blob/master/16.md#regular-events
> A regular event is defined as an event with a kind 1000 <= n < 10000. Upon a regular event being received, the relay SHOULD send it to all clients with a matching filter, and SHOULD store it. New events of the same kind do not affect previous events in any way.